lib/modules/dosomething/dosomething_helpers/dosomething_helpers.module

```diff
@@ -592,3 +592,13 @@ function dosomething_helpers_get_current_language_code() {
 function dosomething_helpers_application_path($path = '') {
   return realpath(DRUPAL_ROOT . '/../' . $path);
 }
+
+/**
+ * Implements hook_form_node_admin_content_alter().
+ */
+function dosomething_global_form_node_admin_content_alter(&$form, &$form_state, $form_id) {
```

Comment:
right...but you have to call the function `dosomething_helpers_form_node_admin_content_alter()` :wink: